src/composables/composablesHelpers.ts

```diff
@@ -15,14 +15,21 @@ export function createNetworkWatcher() {
   let currentNetworkId: string;
 
   return {
-    onNetworkChange: (store: Store<any>, callback: () => void) => {
+    onNetworkChange: (
+      store: Store<any>,
+      // eslint-disable-next-line no-unused-vars
+      callback: (activeNetwork: any) => void,
+      immediate = false,
```

Comment:
I suggest to pass this in a similar way as the Vue `watch` does - as an object of options.